whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapServiceTestIntegration.java

```diff
@@ -1341,8 +1342,166 @@ public void lookup_org_entity_handle() {
                 .get(Entity.class);
 
         assertThat(response.getHandle(), equalTo("ORG-TEST1-TEST"));
+        assertTrue(response.getAutnums().isEmpty());
+        assertTrue(response.getNetworks().isEmpty());
     }
 
+    @Test
```

Comment:
Add a test for references to a *hierarchy* of IPv4 LEGACY resources, only the top-level resource should be returned.